indico/modules/events/contributions/controllers.py

```diff
@@ -147,7 +147,7 @@ def _process(self):
             if tpl_components['hide_contrib']:
                 self.reporter.flash_info_message(contrib)
             return jsonify_data(**tpl_components)
-        return jsonify_form(form)
+        return WPManageContributions.render_template('forms/contribution.html', form=form)
```

Comment:
don't render standalone templates through a WP - use `render_template()` from flask. In case of an ajax request there's not much the WP render_template does anyway